web/src/test/java/com/graphhopper/http/GraphHopperServletIT.java

```diff
@@ -138,6 +140,23 @@ public void testGraphHopperWeb() throws Exception {
     }
 
     @Test
+    public void testNonTranslatedInstructions() {
+        GraphHopperAPI hopper = new GraphHopperWeb();
+        assertTrue(hopper.load(getTestRouteAPIUrl()));
+        GHResponse rsp = hopper.route(new GHRequest(42.554851, 1.536198, 42.510071, 1.548128));
+        assertEquals("Continue onto Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
+
+        GHRequest request = new GHRequest(42.554851, 1.536198, 42.510071, 1.548128);
+        request.getHints().put("translate_instructions", false);
+        rsp = hopper.route(request);
+        assertEquals("Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
```

Comment:
Mhm, not that sure, I think this is exactly the meaning. Maybe we should call it pretranslate_instructions? A client would usually pass this parameter when translating the instructions on the client side. So the client puts either the raw translated name into the instruction name or only the street name. But since we don't set the raw flag we have to translate it on the client side. We only use this parameter on the Java client. I think the server is not evaluating this parameter, since we always return both.